### ISSUE TITLE
How to connect with the .gdb extension database?

### ISSUE THREAD

**Issue by JoseCarlos056** [hgourvest/node-firebird#268]
This is the error returned:
![image](https://user-images.githubusercontent.com/38230621/154699584-042cd05b-de01-4661-822a-4b37e42b7d34.png)
config:
`{
  "host": "127.0.0.1",
  "port": 3050,
  "database": "Conlab.gdb",
  "user": "SYSDBA",
  "password": "masterkey",
  "lowercase_keys": false,
  "role": null,
  "pageSize": 4096,
  "retryConnectionInterval": 1000
}
`
attempt:
`Firebird.attach(firebirdConf, function (err, db) {
    if (err) throw err;
    // db = DATABASE
    db.query('SELECT * FROM TABLE', function (err, result) {
      // IMPORTANT: close the connection
      console.log(result);
      db.detach();
    });
  });`

**Comment by iyudincev** [comment]
Use either a full file path or an alias.
